res/controllers/Pioneer-HID-unlock.hid.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MixxxControllerPreset schemaVersion="1" mixxxVersion="1.11+">
+    <info>
+        <name>Pioneer Hardware Unlock</name>
+        <author>Swiftb0y</author>
+        <description>"Unlocks" some functions pioneer DJ locks behind licensing, supported (but untested) devices: CDJ-2000NXS2, DJM-900NXS2, CDJ-TOUR1, DJM-TOUR1, XDJ-1000MK2, DJM-450, DJM-250MK2, DJM-750MK2, DJM-V10, XDJ-XZ </description>
+        <manual></manual>
+        <devices>
+            <product protocol="hid" vendor_id="0x1157" product_id="0x300" usage_page="0xffa0" usage="0x1" interface_number="0x3" />
```

Comment:
this is just taken from some other mapping. So this definitely needs changing in order to properly work, I think.